packages/proxy/schema/model_list.json

```diff
@@ -3777,7 +3791,7 @@
     "output_cost_per_mil_tokens": 12.0,
     "max_input_tokens": 65536,
     "max_output_tokens": 32768,
-    "parent": "gemini-3-pro-image-preview"
+    "displayName": "Gemini 3 Pro Image Preview"
```

Comment:
not sure why this had parent like this. when an org only has vertex, the option gets removed from the list of models